### PR TITLE
networkd: Permit all-zero RoutingPolicyRule prefixes

### PR DESCRIPTION
**Pull request by hvenev**
For example this `From` address range is no longer ignored:

    [RoutingPolicyRule]
    From=0.0.0.0/8

### CODE REVIEW

File: test/test-network/conf/routing-policy-rule-test1.network [hunk 1]
```diff
@@ -23,3 +24,9 @@ IncomingInterface=test1
 Table=9
 Priority=101
 Family=both
+
+[RoutingPolicyRule]
+IncomingInterface=test1
+From=0.0.0.0/8
+Table=10
```
Comment (yuwata) [inline test/test-network/conf/routing-policy-rule-test1.network:31]: Please add `10` to `routing_policy_rule_tables` array in systemd-networkd-tests.py.

**Review by yuwata — CHANGES_REQUESTED** [review]
LGTM. But one minor thing.

File: test/test-network/conf/routing-policy-rule-test1.network (hunk 1 above)
Comment (hvenev) [reply test/test-network/conf/routing-policy-rule-test1.network:31]: Done

**Comment by yuwata** [comment]
LGTM.

**Pull request merged**